src/lib.rs

```diff
@@ -148,7 +148,7 @@ impl<IO: DfuIo> DfuSansIo<IO> {
 /// DFU Status.
 ///
 /// Note: not the same as state!
-#[derive(Debug, Clone, Copy, PartialEq, Display)]
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Display)]
```

Comment:
It might be worth using repr here? And assigning value to variants?

Reply:
> This looks like excellent work. Is it already ready to merge?

Sure; next steps to follow soon building on top of this one

Reply:
> It might be worth using repr here? And assigning value to variants?

Possible; though not entirely sure what that would bring; Unless e.g. num_derive is brought in as a main dependency to drive a from like trait